docs/deployment_guide/partner_editable/troubleshooting.adoc

```diff
@@ -101,11 +101,11 @@ Verify that an RSA key pair used during deployment. If not, try the following op
 :eks-1_24-cl: https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html#kubernetes-1.24
 :eksqs-5_0_0-cl: https://github.com/aws-quickstart/quickstart-amazon-eks/releases/tag/v5.0.0
 
-=== Version 1.4.1
+=== Version 1.4.2
 * The stack now reports the status of the  {partner-product-short-name} deployment and installation of the sample application  after all of the infrastructure has been deployed.
 * Fixed an issue that occurred when deleting VPCs because of stale security groups.
 * Upgraded the following versions:
-** {tap-1_4_1-cl}[{partner-product-acronym}^] 1.4.1.
+** {tap-1_4_1-cl}[{partner-product-acronym}^] 1.4.2.
```

Comment:
The {tap-1_4_1-cl} is a misnomer.
the variable name has 1.4.1 but the RN link is always for the major version 1.4 and it has all minor versions on the same [page](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.4/tap/release-notes.html#v141-0).